composeApp/src/main/kotlin/io/github/couchtracker/tmdb/TmdbDepartment.kt

```diff
@@ -0,0 +1,26 @@
+package io.github.couchtracker.tmdb
+
+import app.moviebase.tmdb.model.TmdbDepartment
+import io.github.couchtracker.R
+import io.github.couchtracker.utils.Text
+
+@Suppress("CyclomaticComplexMethod")
```

Comment:
Instead of supressing, can you just configure the rule better (e.g. set `ignoreSingleWhenExpression` and/or `ignoreSimpleWhenEntries`).

https://detekt.dev/docs/rules/complexity/#cyclomaticcomplexmethod